packages/select/src/ListMenu/ListMenu.types.ts

```diff
@@ -1,11 +1,8 @@
 import { HTMLElementProps } from '@leafygreen-ui/lib';
-import { PopoverProps } from '@leafygreen-ui/popover';
 
 import { DropdownWidthBasis } from '../Select/Select.types';
 
-export interface ListMenuProps
-  extends HTMLElementProps<'ul', HTMLLIElement>,
-    Omit<PopoverProps, 'active' | 'refEl' | 'onClick'> {
+export interface ListMenuProps extends HTMLElementProps<'ul', HTMLLIElement> {
```

Comment:
Should be `HTMLULElement` (mistake from previous).

Probably cleaner to use `ComponentPropsWithoutRef<'ul'>` though